tensorrt_llm/_torch/models/modeling_deepseekv3.py

```
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: Copyright (c) 2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Ruff (0.16.1)

[warning] The file is executable but no shebang is present

(EXE002)